backend/src/controllers/tagController.ts

```
@@ -115,32 +122,33 @@ export class TagControllers {
-    const {tagStatus} = req.params;
-    if((await TagRepository.findTagByStatus(tagStatus)) === "active" || "inactive"){
+  static getTagByTagStatus = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+  ) => {
+    const { tagStatus } = req.params;
+    if (
+      // (await TagRepository.findTagByStatus(tagStatus)) === "active"
+      (await TagRepository.findOne({ tagStatus: tagStatus })) === "active" ||
+      "inactive"
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)